tests/e2e/util.go

```diff
@@ -1983,8 +1983,8 @@ func bringSvcK8sAPIServerUp(ctx context.Context, client clientset.Interface, pvc
 //pvcHealthAnnotationWatcher polls the health status of pvc and returns error if any
 func pvcHealthAnnotationWatcher(ctx context.Context, client clientset.Interface, pvclaim *v1.PersistentVolumeClaim, healthStatus string) error {
 	framework.Logf("Waiting for health annotation for pvclaim %v", pvclaim.Name)
-	waitErr := wait.Poll(pollTimeoutShort, pollTimeout, func() (bool, error) {
-		framework.Logf("wait for next poll %v", pollTimeoutShort)
+	waitErr := wait.Poll(healthStatusPollTimeout*time.Second, healthStatusPollTimeout*time.Minute, func() (bool, error) {
```

Comment:
Its better to have 2 variables, `healthStatusPollInterva`l and `healthStatusPollTimeout`

Reply:
done